src/js/profile/mobile/widget/DropdownMenu.js

```diff
@@ -1010,17 +1011,19 @@
 					widgetParentStyle = window.getComputedStyle(widgetParent),
 					maxContainerWidth;
 
-				self._offset = getOffsetOfElement(ui.elSelectWrapper, ui.page);
+				self._offsetTop = getOffsetOfElement(ui.elSelectWrapper, ui.page);
 				areaInfo = self._chooseDirection();
 
 				width = biggestListItemWidth > wrapperMinWidth ?
 				biggestListItemWidth + parseInt(stylesOfSelectedOptionAfter.marginRight, 10) +
 				parseInt(stylesOfSelectedOptionAfter.width, 10) : wrapperMinWidth;
 
 				height = optionHeight;
+
 				// This part decides the location and direction of option list.
-				offsetLeft = self._offset.left;
+				offsetLeft = self._horizontalDirection === "right" ? window.screen.width - width : 0;
 				optionStyle = "left: " + offsetLeft + "px; ";
+
 				if (options.inline === true) {
```

Comment:
Type is boolean, do I need "true"? If it can be omitted, nothing is concise.

Reply:
Are you refering to this line?
`if (options.inline === true)`
I am not changing this and could be refactored separately.

Reply:
I agree with Grzegorz. This is out of scope this PR.